gitea/org_team.go

```diff
@@ -7,10 +7,11 @@ package gitea
 
 // Team represents a team in an organization
 type Team struct {
-	ID           int64         `json:"id"`
-	Name         string        `json:"name"`
-	Description  string        `json:"description"`
-	Organization *Organization `json:"organization"`
+	ID                int64         `json:"id"`
+	Name              string        `json:"name"`
+	Description       string        `json:"description"`
+	Organization      *Organization `json:"organization"`
+	IsAllRepositories bool          `json:"is_all_repositories"`
```

Comment:
It would better to call it `IncludeAllRepositories` or something like that. Current name does not sound natural imho

Reply:
You're right. I will rename it.